src/interfaces/coral_web/src/themes/cohereTheme.js

```diff
@@ -9,6 +9,38 @@ module.exports = {
         pureWhite: '#FFFFFF',
         black: '#212121',
         white: '#FAFAFA',
+        vb: {
```

Comment:
what are these color names?

`vb` `mw` `sg` `sr`

Reply:
I asked the same to James, https://www.figma.com/design/nZgO5ONCPP3efPi7VdWuR2?node-id=2383-49161#874267251

Reply:
i think: 
- `vb`: `volcanic-black`
- `mw`: `marble-white`
- `sg`: `success-green`
- `sr`: `success-red`

Reply:
tbh, I like it more `vb` than `volcanic-black` cause otherwise, the classes will be even longer, such as `bg-volcanic-black-500` instead of `bg-vb-500`. Feel free to change it if you like

Reply:
maybe wait for design confirmation first some of the color values clash with what we have. 
ie. we want to update `vocanic-500` to  `#808080` but our current `volcanic-500` is `#9E9E9E` 🤔